docs/content/latest/deploy/public-clouds/aws/manual-deployment.md

```diff
@@ -53,7 +53,7 @@ We now have 2 VMs each in Availability Zones `us-west-2a`, `us-west-2b`, `us-wes
 
 Now that the six nodes have been prepared, the yb-master process will be run on three of these nodes (because RF=3) and yb-tserver will be run on all six nodes. To learn more about YugabyteDB’s process architecture, see [here](../../../../architecture/concepts/universe/).
```

Comment:
change `process architecture` to `server architecture`

Reply:
fixed